src/actions/passphrase.js

```diff
@@ -35,7 +35,10 @@ const defaultErrorAction = {
 export const getErrorDetails = error => {
   const vaultError = error && error.response && error.response.error
   const stackError =
-    error && error.errors && error.errors[0] && error.errors[0].detail
+    error &&
+    error.reason.errors &&
+    error.reason.errors[0] &&
+    error.reason.errors[0].detail
```

Comment:
nit: Seems like this project now supports the `?.` operator, so `const stackError = error?.reason?.error?.[0].detail` should work

Reply:
Yes way nicer 👌